### DISCUSSION TITLE
How can synthea generate shorter data? The current data is too long to import into the database.

### DISCUSSION THREAD

**Discussion by QzZhang3** [synthetichealth/synthea#1361] [General]
How can synthea generate shorter data? The current data is too long to import into the database.

**Comment by jawalonoski** [comment]
By default Synthea exports the last 10 years of data for each patient.

You can change this on the command line or in the `synthea.properties` file:

Command line example for 1 year of history:
```
./run_synthea --exporter.years_of_history=1
```

See:
https://github.com/synthetichealth/synthea/blob/3c23908e922534b5ce2b1ccca3782f84017b2a09/src/main/resources/synthea.properties#L8-L10
